src/pages/api/getContributors.ts

```diff
@@ -0,0 +1,32 @@
+import { UserModel } from '@server/db/user'
+import { apiController } from '@utils/api'
+
+export interface GetContributorsParams {
```

Comment:
this API seems similar to `LoadUserInfo` we have right now. Could you try merge the two into one? Or change the api to take question id and provide the info about the contributors of the question

Reply:
> this API seems similar to `LoadUserInfo` we have right now. Could you try merge the two into one? Or change the api to take question id and provide the info about the contributors of the question

I have made it so that the api gets qid and returns the contributors here https://github.com/kixlab/kuiz/pull/52/commits/fa45c8f6d662ed64f784f2b2d14a979e2d96556e :)